hack/vendor-console.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o errexit 
+set -o nounset 
+set -o pipefail
+
+STARTTIME=$(date +%s)
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
```

Comment:
I think you probably want to have stack traces so errors are easier to understand?
```sh
source "${OS_ROOT}/hack/util.sh"
os::log::install_errexit
```